plugins/plugin-chart-echarts/src/Timeseries/types.ts

```diff
@@ -116,12 +116,13 @@ export interface EchartsTimeseriesChartProps extends ChartProps {
 }
 
 export interface TimeseriesChartTransformedProps {
-  formData: EchartsTimeseriesFormData;
-  height: number;
-  width: number;
   echartOptions: EChartsOption;
   emitFilter: boolean;
-  setDataMask: SetDataMaskHook;
-  labelMap: Record<string, DataRecordValue[]>;
+  formData: EchartsTimeseriesFormData;
```

Comment:
This part of the type seems to be the same in every echarts except the formdata declaration, so I abstracted an interface in `plugin-chart-echarts/src/types.ts`,  maybe we can reuse it?

Reply:
Great improvement! 👍